JobConfig/primary/IPAMuminusMichel.fcl

```diff
@@ -6,16 +6,57 @@
 #
 #include "Production/JobConfig/primary/IPAStopParticle.fcl"
 
-physics.filters.PrimaryFilter.MinimumPartMom : 30.0 // MeV/c TODO - tweak this parameter to get more DetSteps
+physics.filters.PrimaryFilter.MinimumPartMom : 1.0
 physics.producers.FindMCPrimary.PrimaryProcess : "mu2eMuonDecayAtRest"
 
-physics.producers.generate : {
-  module_type : Pileup
-  verbosity : 0
-  inputSimParticles: IPAMuminusStopResampler
-  captureProducts : []
-  decayProducts : [ @local::Pileup.dioGenTool ]
-  stoppingTargetMaterial : "IPA"
+CompositeGenerator: {
+  module_type: CompositeMaterialGenerator
+  tag: IPAMuminusStopResampler
+  process: "mu2eMuonDecayAtRest"
+  weighting: {
+    tool_type: AtomicVolumeSamplerTool
+    material: "IPAPolyethylene"
+  }
+  elements: [
+    {
+      name: "C"
+      position_tool: {
+        tool_type: MuStopDecayPositionSamplerTool
+        atom: "C"
+      }
+      generator_tool: {
+        tool_type: DIOGenerator
+        spectrum: {
+          pdgId: 11
+          spectrumShape: "tabulated"
+          spectrumVariable: "totalEnergy"
+          spectrumFileName: "Offline/ConditionsService/data/heeck_finer_binning_2016_szafron-scaled-to-6C.tbl"
+          elow: 1.0
+          ehi: 110.0
+        }
+      }
+    },
+    {
+      name: "H"
+      position_tool: {
+        tool_type: MuStopDecayPositionSamplerTool
+        atom: "H"
+      }
+      generator_tool: {
+        tool_type: DIOGenerator
+        spectrum: {
+          pdgId: 11
```

Comment:
Does your code read this or someone else's?  

If it's your code please use the name of the particle, not it's number.   You can construct an object of type PDGCode from a string and it will scan the known strings to find the number.

Reply:
The blocks containing the `pdgId` fields are not read by the new generator --- they are only seen as `fhicl::DelegatedParameter`s at the generator level. The code which reads the `pdgId` field here is [the preexisting DIOGenerator tool](https://github.com/Mu2e/Offline/blob/main/EventGenerator/src/DIOGenerator_tool.cc).

Reply:
Thanks.  This is distant enough from the purpose of this PR that we can defer the change to a later PR